compiler/src/iree/compiler/Dialect/HAL/IR/HALBase.td

```diff
@@ -689,6 +689,31 @@ def HAL_DeviceTargetAttr :
     // the `hal.device.targets` attribute is found.
     static SmallVector<DeviceTargetAttr, 4> lookup(Operation *op);
 
+    // Returns true if there is any UnitAttr with |name| in any device
+    // configuration for the given |op|.
+    static bool getConfigAttrAny(Operation *op, StringRef name);
```

Comment:
Funnily we were discussing naming prefix for bool functions, coming to should is or has be the prefix. Java apparently is always is, in the current function has reads better ... But today we are using get even for bool.

Reply:
yeah, lots of potential bike-shedding :) I went with gets as these are not accessors but lookups/gathers/etc - could prefix them all lookup to make them consistent, though, so I'll do that. Having different prefixes is hard because english doesn't really like the ordering for things (hasConfigAttrAny doesn't sound good, hasAnyConfigAttr sounds like it's doing a set check, haveAllConfigAttrs/haveConfigAttrAll/etc when plural gets weird, etc).